test/integration_test.sh

```diff
@@ -80,80 +87,135 @@ ss2_version=${10}
 work_dir=$(pwd)
 script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-# 1. Clone Lira if needed 
-# TODO: Create a repo for integration_test.sh to go in, so that we
-# can run in github mode. Currently we have to run in lira local mode.
-lira_dir=$script_dir/../
+# 1. Clone Lira if needed
+if [ $lira_mode == "github" ]; then
+  git clone git@github.com:HumanCellAtlas/lira.git
+  lira_dir=lira
+  cd $lira_dir
+  if [ $lira_version == "latest_released" ]; then
+    lira_version=$(python $script_dir/get_latest_release.py HumanCellAtlas/lira)
+  elif [ $lira_version == "latest_deployed" ]; then
+    lira_version=$(python $script_dir/current_deployed_version.py lira)
+  fi
+  git checkout $lira_version
+  cd -
+elif [ $lira_mode == "local" ]; then
+  lira_dir=$lira_version
+fi
 
 # 2. Get pipeline-tools version
-# TODO: Uncomment condition below once we have Lira reading wrapper WDLs from pipeline-tools repo
-#if [ $pipeline_tools_mode == "github" ] && [ $pipeline_tools_version == "current" ]; then
-  pipeline_tools_version=$(python $script_dir/current_deployed_pipeline_version.py \
-                    --mint_deployment_dir $mint_deployment_dir \
-                    --env $env \
-                    --pipeline_name pipeline_tools)
-  echo "pipeline_tools_version: $pipeline_tools_version"
-#fi
+if [ $pipeline_tools_mode == "github" ]; then
+  if [ $pipeline_tools_version == "latest_deployed" ]; then
+    pipeline_tools_version=$(python $script_dir/current_deployed_version.py \
+                      --mint_deployment_dir $mint_deployment_dir \
+                      --env $env \
+                      --component_name pipeline_tools)
+    echo "pipeline_tools_version: $pipeline_tools_version"
+  fi
+  pipeline_tools_prefix="https://raw.githubusercontent.com/HumanCellAtlas/pipeline-tools/${pipeline_tools_version}"
+elif [ $pipeline_tools_mode == "local" ]; then
+  pipeline_tools_prefix="/pipeline-tools"
+  pipeline_tools_dir=$pipeline_tools_version
+  cd $pipeline_tools_dir
+  pipeline_tools_dir=$(pwd)
+  cd -
+fi
 
 # 3. Build or pull Lira image
 if [ $lira_mode == "image" ]; then
-  if [ $lira_version == "latest" ]; then
-    lira_image_version=$(python $script_dir/get_latest_release.py HumanCellAtlas/secondary-analysis)
+  if [ $lira_version == "latest_released" ]; then
+    lira_image_version=$(python $script_dir/get_latest_release.py HumanCellAtlas/lira)
+  elif [ $lira_version == "latest_deployed" ]; then
+    lira_image_version=$(python $script_dir/current_deployed_version.py lira)
   else
     lira_image_version=$lira_version
   fi
-  docker pull humancellatlas/secondary-analysis:$lira_image_version
+  echo "Not pulling lira image -- fix this"
+  #docker pull humancellatlas/lira:$lira_image_version
```

Comment:
Currently, the image of lira(listener) is sitting on the gcr.io, if we don't have any credentials in the image, we could move it out. However, I have two thoughts here:
1. Do we really want to move this image out from the gcr? Since lira is deployed on gke, for this specific image, do we benefit from letting it sit on the gcr? The shortcoming is our docker images would be decentralized then, and not easy to manage.
2. If we want to move the docker image out from the gcr, should we wait for few days and move it directly to quay? My personal preference is not expanding our dockerhub images anymore before we migrating to quay.